src/main/java/vic/cinema/model/CinemaHall.java

```diff
@@ -0,0 +1,47 @@
+package vic.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cinema_hall")
```

Comment:
```suggestion
@Table(name = "cinema_halls")
```
якщо я правильно запам"ятав, то коректно називати таблиці у множині

Reply:
Done